.github/workflows/ci.yml

```
@@ -0,0 +1,52 @@
+  pull_request:
+    branches:
+      - main
+      - dev
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    # 1. 코드 체크아웃
+    - name: Checkout code
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)